packages/openapi-typescript/test/schema-object.test.ts

```diff
@@ -278,9 +278,19 @@ describe("Schema Object", () => {
         expect(generated).toBe("OneOf<[string, boolean, number, null]>");
       });
 
-      test("enum + polymorphism + nullable", () => {
-        const generated = transformSchemaObject({ type: ["string", "null"], enum: ["", "false positive", "won't fix", "used in tests"] }, options);
-        expect(generated).toBe(`"" | "false positive" | "won't fix" | "used in tests" | null`);
```

Comment:
Yeah thanks for changing these. I don’t think the old strings were meaningful / helpful.